demo/xcb_cairo/main.c

```diff
@@ -0,0 +1,159 @@
+/* nuklear - v1.32.0 - public domain */
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+/*#include <limits.h>*/
+/*#include <math.h>*/
+/*#include <sys/time.h>*/
+/*#include <unistd.h>*/
+/*#include <time.h>*/
```

Comment:
Reason why these a just commented out? Do the need to be removed or+

Reply:
Actually I don't remember, I usually try to reduce the amount of includes as most as possible. It works without them but was included in x11 demo... I will remove the lines